reference/supported-chains.md

```diff
@@ -9,84 +9,60 @@ description: >-
 ### POKT Mainnet
 
 Below is the full list of supported, reward-generating RelayChains, identical to the `SupportedBlockchains` parameter.
+**NOTE:** As of December 18, 2024, all services have **one** ServiceID for both mainnet and archival nodes. The legacy ServiceIDs will no longer be valid. If you were previously using separate IDs for mainnet and archival, please update to the new IDs listed below. Ethereum is the one exception as the new Service ID includes what is previously known as mainnet, archival and trace services. 
+
+| Name                                       | ServiceID | Legacy ServiceID |
+| ------------------------------------------ | --------- | ---------------- |
+| Arbitrum One                               | F001      | 0066             |
+| Arbitrum Sepolia Testnet                   | F002      | A086             |
+| Avalanche                                  | F003      | 0003, A003       |
+| Avalanche DFK                              | F004      | 03DF             |
+| Base                                       | F005      | 0079             |
+| Base Sepolia Testnet                       | F006      | 0080             |
+| Bitcoin                                    | F007      | N/A              |
+| Blast                                      | F008      | A090             |
+| BNB Chain                                  | F009      | 0004, 0010       |
+| Boba                                       | F00A      | 0048             |
+| Celestia Consensus                         | A0CB      | A0CB             |
```

Comment:
Should we write No Change on Legacy Server ID for the Celestia ones?

Reply:
Where were you thinking? in the notes at the top?

Reply:
Maybe leave an asterisk on each of those and a comment below.

Also, while you're in here, you should make a line item to validate separately what the testnet supports, because changes have been made to those variables and I'm not sure which NRs are supporting any traffic to full nodes on testnet.

Reply:
Fixed - I updated the Celestia & Ethereum IDs with asterisks and add the footnote at the bottom of the table. I also updated the Testnet table to remove chains that are no longer supported by the network.